UI/Web/src/app/cards/card-item/card-item.component.html

```diff
@@ -44,6 +44,9 @@
             <i class="fa {{format | mangaFormatIcon}}" aria-hidden="true" *ngIf="format != MangaFormat.UNKNOWN" title="{{formatString}}"></i>
             <span class="visually-hidden">{{formatString}}</span>
           </ng-container>
+          <ng-container *ngIf="volumeTitle != ''">
```

Comment:
Remove this code. It adds complexity to the card. The card is assumed that it will render out the title as is, not have the card add other things randomly.